.github/workflows/publish-docker-image.yml

```diff
@@ -55,4 +56,4 @@ jobs:
         platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
         # Only push if we are publishing a release
         push: true
-        tags: testcontainers/ryuk:${{ github.event.release.tag_name }}
+        tags: us-docker.pkg.dev/testcontainers-oss/testcontainers/ryuk:${{ github.event.release.tag_name }}
```

Comment:
Should we tag both or only the new one?

Reply:
I am fine pushing older versions manual. Anything else would be a bit difficult (recreating the release), or do you have anything specific in mind? Is `0.4.0` already used in other language implementations? .NET still use `0.3.4` in case developers have issues with the new version it might be a good idea to push the `0.3.4` too.

Reply:
existing images were moved manually using [crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane_copy.md)